### PR TITLE
Added repl funcionality

### PR DESCRIPTION
**Pull request by wastella**
I made the repl for the calculator, now I just need to make the pypi functionality.

### CODE REVIEW

File: calculator/src/cli.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/usr/local/bin/python3
```
Comment (cestella) [inline calculator/src/cli.py]: This should be:
```
#!/usr/bin/env python3
```
see [here](https://stackoverflow.com/questions/6908143/should-i-put-shebang-in-python-scripts-and-what-form-should-it-take) for a discussion

File: calculator/src/cli.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+from src.lexer import lex
+from src.parser import rpn, infix
+
+
+def main():
```
Comment (cestella) [inline calculator/src/cli.py:12]: This looks good, but I'd like it if you could use the calculator by piping input into it from another program.  For instance, in the shell, can you run:
```
echo "1 + 1" | calc
```
and have it print out `2`?

**Review by cestella — CHANGES_REQUESTED** [review]
One VERY small issue and it's good to go!

File: calculator/src/cli.py (hunk 2 above)
Comment (cestella) [reply calculator/src/cli.py:12]: It appears this isn't going to work after looking at it with you.  I want you to do the following:
1. Catch the `EOFError` that is thrown by `input()`
2. Use [argparse](https://docs.python.org/3/library/argparse.html) to allow `-q` or `--quiet` in which you do not provide the prompt.

File: calculator/src/cli.py (hunk 1 above)
Comment (wastella) [reply calculator/src/cli.py]: 👍

File: calculator/src/cli.py (hunk 2 above)
Comment (wastella) [reply calculator/src/cli.py:12]: 👍

**Review by cestella — APPROVED** [review]
LGTM!

**Pull request merged**